test/unit/org/apache/cassandra/cql3/validation/entities/CollectionsTest.java

```diff
@@ -38,7 +38,7 @@
 public class CollectionsTest extends CQLTester
 {
     @Test
-    public void testMapBulkRemoval() throws Throwable
```

Comment:
@Maxwell-Guo please return all the changes in this file. The changes here have nothing to do with your patch.